container-se-managed/src/main/java/org/jboss/arquillian/container/se/managed/ManagedSEDeployableContainer.java

```diff
@@ -209,14 +211,14 @@ public ProtocolMetaData deploy(final Archive<?> archive) throws DeploymentExcept
             throw new DeploymentException("Could not start process", e);
         }
 
-        int waitTime = debugModeEnabled ? 15 : 5;
+        int finalWaitTime = debugModeEnabled ? 15 : waitTime;
```

Comment:
Maybe we should also change the debug value so that it corresponds to the wait time value. E.g. `3 * waitTime`?